localEvaluation/localEvaluation.go

```diff
@@ -127,71 +112,57 @@ func fetch(flagKeys []string, user UserProperties, valueOnly bool) map[string]in
 
 	result, err := client.EvaluateByOrg(&expUser)
 	if err != nil {
-		return variants
+		return nil, err
 	}
-	filter := len(flagKeys) != 0
-	for k, v := range *result {
-		if v.IsDefaultVariant {
-			continue
-		}
-		if !filter {
-			if valueOnly {
-				variants[k] = v.Variant.Key
-			} else {
-				variants[k] = AmplitudeVariant{
-					Value:   v.Variant.Key,
-					Payload: v.Variant.Payload,
-				}
-			}
-			continue
+	return result, nil
+}
+
+func getValue(flagName string, user UserProperties) local.EvaluationVariant {
+	result, _ := fetch(user)
+	if *result != nil {
```

Comment:
fetch func returns pointer to EvaluationResult. So should this check be 
if result != nil   ? 
because dereferencing will cause NPE.